test/crit/Makefile

```diff
@@ -11,25 +11,53 @@ e2e-test: ../../crit/bin/crit test-imgs crit-test.sh
 	@echo "Running E2E test"
 	@./crit-test.sh
 
-test-imgs: ../loop/loop
-	$(eval PID := $(shell ../loop/loop))
-	mkdir -p $@
-	$(CRIU) dump -v4 -o dump.log -D $@ -t $(PID)
-	$(CRIU) restore -v4 -o restore.log -D $@ -d
-	cp /proc/${PID}/environ $@
-	cp /proc/${PID}/cmdline $@
+define gen_mmapper_imgs
+	$(eval MM_PID := $(shell ../mmapper/mmapper $(1) $(2) /tmp/mmapper-${1}-${2}.log))
+	$(eval MM_DIR := $(3))
+	mkdir -p $(MM_DIR)
+	cp /proc/$(MM_PID)/environ $(MM_DIR)
+	cp /proc/$(MM_PID)/cmdline $(MM_DIR)
+	$(CRIU) dump -v4 -o dump.log -D $(MM_DIR) -t $(MM_PID)
+	$(CRIU) restore -v4 -o restore.log -D $(MM_DIR) -d
+endef
+
+test-imgs: ../loop/loop ../mmapper/mmapper
+	$(eval LOOP_PID := $(shell ../loop/loop))
+	$(eval LOOP_DIR := $@/loop)
+	mkdir -p ${LOOP_DIR}
+	cp /proc/${LOOP_PID}/environ ${LOOP_DIR}
+	cp /proc/${LOOP_PID}/cmdline ${LOOP_DIR}
+	$(CRIU) dump -v4 -o dump.log -D ${LOOP_DIR} -t $(LOOP_PID)
+	$(CRIU) restore -v4 -o restore.log -D ${LOOP_DIR} -d
 	pkill -9 loop
 
+	$(call gen_mmapper_imgs, 257, p, $@/mm_p_257b)
```

Comment:
It would be good to add a comment here. Something like
```Makefile
# gen_mmapper_imgs <memory size> <mmap flags> <images dir>
# p - private; s - shared; a - annonymous
```